public/views/projects-migrated.html

```diff
@@ -39,10 +33,22 @@ <h4 class="site-header">{{migratedProject.site_name}}</h4>
 				 <span ng-show="migratedProject.destination_url===null">Exported to <span ng-if="migratedProject.destination_type==='mailarchivezip'">zip</span><span ng-if="migratedProject.destination_type!=='mailarchivezip'">{{migratedProject.destination_type}}</span></span>
          <span ng-show="migratedProject.migrated_by"> by {{migratedProject.migrated_by}}</span>
 				</div>
-        <div class="col-md-6 col-lg-6 col-sm-6 col-xs-6" style="text-align:right">
-          <small ng-show="migratedProject.status.data.counts.failures !==0" class="export-fail">
-            There were {{migratedProject.status.data.counts.failures}} failures
+        <div class="col-md-6 col-lg-6 col-sm-6 col-xs-6 {{migratedProject.status.status}}" style="text-align:right">
+          <small ng-show="migratedProject.status.counts.successes ===0 && migratedProject.status.counts.errors ===0">
```

Comment:
I think you should be checking for the `partial_successes` value as well. For example, for mbox download if for some reason the attachments URL's from ctools couldn't be resolved then we just drop the attachments but will still process the mail. In that case we will `partial_successes' populated.